acceptance/acceptance_test.go

```diff
@@ -486,3 +490,16 @@ func ListDir(t *testing.T, src string) []string {
 	}
 	return files
 }
+
+func getUVDefaultCacheDir(t *testing.T) string {
+	// According to uv docs https://docs.astral.sh/uv/concepts/cache/#caching-in-continuous-integration
+	// the default cache directory is
+	// "A system-appropriate cache directory, e.g., $XDG_CACHE_HOME/uv or $HOME/.cache/uv on Unix and %LOCALAPPDATA%\uv\cache on Windows"
```

Comment:
Why is the "system-appropriate cache directory" not picked up automatically?

Reply:
Because we set HOME in acceptance_test.go (to prevent tests from picking up user's credentials for databricks)

Reply:
Can you include this in the comment? Useful context.